### ISSUE TITLE
Add JWT token invalidation and refreshing example

### ISSUE THREAD

**Issue by boroda4436** [jloisel/securing-rest-api-spring-security#2]
It will be useful to add token invalidation and/or refreshing examples as it is quite difficult to find a good example on the Internet

**Comment by jloisel** [comment]
Yes, could be interesting. The repository is open to pull requests if you want to contribute to it!
